newrelic/structures_newrelic_one_dashboard.go

```diff
@@ -835,6 +836,10 @@ func flattenVariableNRQLQuery(in *entities.DashboardVariableNRQLQuery) []interfa
 	return out
 }
 
+//func hashString(v interface{}) int {
```

Comment:
(b)

Guessing the cause of this new problem is that we don't convert the list of line_widgets back into a set in the flatten functions of read, I tried doing so multifold, but haven't been able to land at a solution and am still running into panics. The changes below show my most recent try to convert the list of line widgets into a set, and the error I see is 

`panic: interface conversion: interface {} is map[string]interface {}, not string`